src/lib/AnnotationControls.ts

```diff
@@ -160,5 +172,20 @@ export default class AnnotationControls {
         );
 
         regionButton.setAttribute('data-testid', 'bp-AnnotationsControls-regionBtn');
+        regionButton.onkeydown = (event: KeyboardEvent): void => {
```

Comment:
We should be using `addEventListener` rather than raw attributes. Should we be listening at the document level?